style/style.css

```diff
@@ -305,7 +328,7 @@ html{
   
   .accordion-content {
     padding: 0 20px;
-    margin-left: 100px;
+    margin-left: 40px;
```

Comment:
Don't edit things not related to your issue xD, they might be getting fixed in another issue.

Reply:
Okay, have set it back to as it was before.